modules/@apostrophecms/doc-type/ui/apos/components/AposDocEditor.vue

```diff
@@ -619,7 +667,8 @@ export default {
         }
         apos.bus.$emit('content-changed', {
           doc,
-          action: (requestMethod === apos.http.put) ? 'update' : 'insert'
+          action: (requestMethod === apos.http.put) ? 'update' : 'insert',
+          isLocalizing: this.isLocalizing
```

Comment:
I need to pass this new data to all docs `content-changed` emitted events to avoid `TheAposContextBar.vue` updating the window locale....